src/main/java/org/zalando/nakadi/controller/EventPublishingController.java

```diff
@@ -99,6 +104,21 @@ private ResponseEntity postEventInternal(final String eventTypeName,
         }
     }
 
+    private void reportMetrics(final EventTypeMetrics eventTypeMetrics, final EventPublishResult result,
+                               final String eventsAsString, final int eventCount) {
+        if (result.getStatus() == EventPublishingStatus.SUBMITTED) {
+            eventTypeMetrics.reportSizing(eventCount, eventsAsString.getBytes(StandardCharsets.UTF_8).length - 2);
```

Comment:
@rcillo  There are one less comma than the number of JSONObjects in the JSONArray created a few lines earlier, so we can rely on that, I think. The problem is that white spaces, tabs, carriage returns, etc., between objects, will be counted, and they shouldn't.